specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/stable/2025-05-01/ResourceHealth.json

```diff
@@ -1677,6 +1764,10 @@
           "items": {
             "type": "string"
           }
+        },
```

Comment:
should this be an enum? or number? How do people know what a lower number is, if its a string??

Reply:
It's a number but in a string format. If the recommendation is to represent numbers as Int type, we can do that. For example, currently it is "0".

Reply:
changed this to 'integer'